### PR TITLE
Integrate multipoint connection demo as main demo application

### PR DESCRIPTION
**Pull request by jamoralp**

### CODE REVIEW

**Comment by pablogs9** [comment]
Are there some other branch/repo that is related to this?

**Comment by jamoralp** [comment]
> Are there some other branch/repo that is related to this?

Yes: https://github.com/micro-ROS/freertos_apps/pull/4 including changes for example application with dual transport communication, which have just been merged. This PR is ready for revision/integration now.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,12 @@
...
+crazyflie-lib-python
+crazyflie2-nrf-firmware
+multi-git-status
+uros_ws
+.devcontainer
+.vscode
+uros_ws_workingstate
+crazyradio-firmware
+crazyflie2-stm-bootloader
+crazyflie-firmware
```
Comment (pablogs9) [inline .gitignore]: Is this required?

Comment (jamoralp) [reply .gitignore]: I really don't know... Didn't notice about this as it was present previous to my changes. It probably is not required anymore.

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**